pgcommitfest/commitfest/views.py

```diff
@@ -86,14 +86,36 @@ def me(request):
     if patch_list.redirect:
         return patch_list.redirect
 
+    # Get stats related to user for current commitfest
+    curs = connection.cursor()
+    curs.execute(
+        """SELECT
+            ps.status, ps.statusstring, count(*)
+        FROM commitfest_patchoncommitfest poc
+        INNER JOIN commitfest_patchstatus ps ON ps.status=poc.status
+        LEFT JOIN commitfest_patch_authors pa ON pa.patch_id=poc.patch_id
+        LEFT JOIN commitfest_patch_reviewers pr ON pr.patch_id=poc.patch_id
+        LEFT JOIN commitfest_patch_subscribers psubs ON psubs.patch_id=poc.patch_id
+        WHERE commitfest_id=%(id)s 
```

Comment:
This check should be removed, right now we don't filter the list by commitfest_id. So the counts shouldn't be filtered by it either.